VocalRaidAssistant.lua

```diff
@@ -118,11 +115,14 @@ function VRA:COMBAT_LOG_EVENT_UNFILTERED(event)
 	local timestamp, event, hideCaster, sourceGUID, sourceName, sourceFlags, sourceFlags2, destGUID, destName,
 		destFlags, destFlags2, spellID, spellName = CombatLogGetCurrentEventInfo()
 
+	-- apply spell correction (e.g. hex and polymorh change the spellID)
+	spellID = addon.spellCorrections[spellID] or spellID
```

Comment:
This is clever, look up the spellID in the correctionList, if nil we ignore it and just use the regular spellID for lookups.